mergify_engine/gh_pr_fullifier.py

```diff
@@ -114,11 +108,18 @@ def disabled_by_rules(pull, **extra):
 
 def compute_weight_and_status(pull, **extra):
     status_desc = disabled_by_rules(pull, **extra)
+    reviews_ok, reviews_ko, reviews_required = pull.mergify_engine["approvals"]
```

Comment:
I just remember that redis may have the old version with 4 items in the tuple.

Reply:
Do you mean a currently running Redis, or something else here in the code?
I imagine you can empty the cache?

Reply:
The currently running redis.

I could empty the cache. But currently clearing the cache have one downside:

When you receive the next "pull request closed/merged" event, mergify-engine will remove this PR from the cache and then looks into the cache to choice the next PR to merge or rebase. If I clear it nothing will happen.

The workaround is the run "refresh.sh" on all installations, but that kind of exhaust github ratelimit on projects with a lot of opened pull requests.

To resume, this cache is a bit more than a cache ;)

Reply:
I push https://github.com/Mergifyio/mergify-engine/commit/55a2ff009befede6d4087ee5221a2ae520d6d6ad to invalidate the cache when needed.